ipatests/test_xmlrpc/tracker/certmapdata.py

```diff
@@ -50,11 +52,9 @@ def _data_from_options(self, **options):
                     certs = [certs]
 
             for cert in certs:
-                cert = x509.load_pem_x509_certificate(
-                    (b'-----BEGIN CERTIFICATE-----\n'
-                     b'{}-----END CERTIFICATE-----\n'
-                     .format(cert)),
-                    default_backend()
+                cert = x509.load_der_x509_certificate(
```

Comment:
The cert is built with a PEM format in ipatests/test_xmlrpc/test_certmap_plugin.py, line 148:  u'certificate': (u'MIICwzCCAau...'), so you probably don't need this change.

Reply:
In that case I would not understand why it was previously built like that. I can try to load `cert` as PEM right away, but applying `.format()` on bytes won't work so at least minor change is needed.

Reply:
Hmm, I see you write `u'certificate': (u'MIICwzCCAau...'),`. That, however, is not a PEM formatted cert AFAIK because it's missing the required header/footer. It's just a base64-encoded DER cert.

Do you propose not to convert the cert to cryptography `Certificate` at all? I suppose that as well might work.

Reply:
Hi @stlaz,
I had missed the point about format() on bytes, sorry about that. You are right, your proposal from the PR should work (ie apply base64.d64decode).